x/concentrated-liquidity/pool_test.go

```diff
@@ -80,6 +80,11 @@ func (s *KeeperTestSuite) TestInitializePool() {
 			if test.expectedErr == nil {
 				// Ensure no error is returned
 				s.Require().NoError(err)
+
+				// ensure that fee accumulator has been properly initialized
+				feeAccumulator, err := s.App.ConcentratedLiquidityKeeper.GetFeeAccumulator(s.Ctx, test.poolI.GetId())
+				s.Require().NoError(err)
+				s.Require().Equal(sdk.DecCoins(nil), feeAccumulator.GetValue())
```

Comment:
this is sdk.DecCoins(nil) not sdk.NewDecCoins() since marshalling and the unmarshalling empty DecCoins struct fills it with nil value

Reply:
nit: we can define a test-global variable like [this](https://github.com/osmosis-labs/osmosis/blob/4a81ee507561a5296fdeaf5f6905dd030d36d9c8/osmoutils/accum/accum_test.go#L37)

I expect this will be frequently used in tests

Reply:
Agreed that using test global variables are way better design. Happy to address this for future PRs and further unnecessary variable duplications